ClearKeep/Scenes/ChangePassword/Views/ChangePasswordContentView.swift

```diff
@@ -24,27 +24,18 @@ struct ChangePasswordContentView: View {
 	@Environment(\.colorScheme) var colorScheme
 	@Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
 	@Environment(\.injected) private var injected: DIContainer
-	@State private(set) var currentPassword: String
-	@State private(set) var newPassword: String
-	@State private(set) var confirmPassword: String
-	@State private(set) var currentStyle: TextInputStyle
-	@State private(set) var newStyle: TextInputStyle
-	@State private(set) var confirmStyle: TextInputStyle
+	@State private(set) var preAccessToken: String = ""
+	@State private(set) var email: String = ""
+	@State private(set) var domain: String = ""
+	@State private(set) var currentPassword: String = ""
+	@State private(set) var newPassword: String = ""
+	@State private(set) var confirmPassword: String = ""
+	@State private(set) var currentStyle: TextInputStyle = .default
+	@State private(set) var newStyle: TextInputStyle = .default
+	@State private(set) var confirmStyle: TextInputStyle = .default
 	
 	// MARK: - Init
-	init(currentPassword: String = "",
-		 newPassword: String = "",
-		 confirmPassword: String = "",
-		 currentStyle: TextInputStyle = .default,
-		 newStyle: TextInputStyle = .default,
-		 confirmStyle: TextInputStyle = .default) {
-		self._currentPassword = .init(initialValue: currentPassword)
-		self._newPassword = .init(initialValue: newPassword)
-		self._confirmPassword = .init(initialValue: confirmPassword)
-		self._currentStyle = .init(initialValue: currentStyle)
-		self._newStyle = .init(initialValue: newStyle)
-		self._confirmStyle = .init(initialValue: confirmStyle)
-	}
+
 	
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/b36012346513a751b0300c2fef0df2c981ef822a/ClearKeep/Scenes/ChangePassword/Views/ChangePasswordContentView.swift#L39"></span>Limit vertical whitespace to a single empty line. Currently 2.<br />
<code>vertical_whitespace</code> <code>ChangePasswordContentView.swift:39</code></td>
    </tr>
  </tbody>
</table>

Reply:
fix complete